frontend/src/views/camp/Dashboard.vue

```diff
@@ -25,10 +25,13 @@
           class="mx-4 mt-6 mb-3"
           style="border-collapse: collapse"
         >
-          <caption class="font-weight-bold text-left">
-            {{
-              periods[uri].description
-            }}
+          <caption class="text-left">
+            <router-link
```

Comment:
I can't seem to focus this link with the keyboard (by tabbing), why is that?

Reply:
Can't reproduce your problem

Reply:
Hm, it only happens on Firefox for me... Let's leave it for now